solver/pb/ops.proto

```diff
@@ -81,6 +81,7 @@ message Mount {
 	CacheOpt cacheOpt = 20;
 	SecretOpt secretOpt = 21;
 	SSHOpt SSHOpt = 22;
+	string inputID = 23;
```

Comment:
Not sure if `inputID` is best name as confuses with the input field . Maybe resultID, refID?